Gemfile

```diff
@@ -37,7 +37,7 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
-gem 'rubocop', '~> 0.55.0', require: false
+gem 'rubocop', '~> 0.56.0', require: false
```

Comment:
trong rails toàn bộ dùng nháy đôi nhé. Em sửa toàn bộ file này từ nháy đơn sang nháy đôi nhé em.
Ngoài ra nên xóa mấy cái comment đi em, nhìn rối code lắm, lại tốn line